src/plone/restapi/services/copymove/copymove.py

```diff
@@ -25,12 +23,8 @@ def __init__(self, context, request):
         self.catalog = getToolByName(self.context, 'portal_catalog')
 
     def get_object(self, key):
-        """Get an object by intid, url, path or UID."""
```

Comment:
We could do a conditional import here, to still have intid support if it's available. But I'm also fine with just dropping it.